README.md

```diff
@@ -1,21 +1,24 @@
 # Capstone_Appointment_back_end
 
-> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface. This app uses RoR, RSPec, Capybara, Devise, and CanCanCan.
+> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface.
 
 ## Built With
 
 - Ruby On Rails (RoR)
 - PostegreSQL
 - RSpec
-- Capybara
-- Devise
-- CanCanCan
-- Bullet
+- Rswag
+- jwt
+- Bcrypt
+- rack-cors and
+- image-processing
 
 ## Live Demo (if available)
 
 [Live Demo Link]()
-
+[API Documentation](https://motocycle-booking.herokuapp.com/api-docs/)
+[Front-end link](https://final-capstone-project-udqc.vercel.app/splash_page)
+ 
```

Comment:
- [ ] In your install instructions, kindly add the `rails db:migrate` and `rails db:seed` that your users will need to install and run your app.